caiman_napari/core.py

```diff
@@ -97,6 +97,8 @@ def add_item(self, algo: str, input_movie_path: str, params: dict):
         # Save DataFrame to disk
         self._df.to_pickle(self.path)
 
+    def remove_item(self, index):
+        self._df.drop(index=index)
```

Comment:
I think you'll also need a `reset_index=True` kwarg to `drop()` . Test this out and see how the indexing works!

Reply:
Modified, appears to be working now!